cmd/soroban-rpc/internal/db/event.go

```diff
@@ -42,9 +55,9 @@ func (eventHandler *eventHandler) InsertEvents(lcm xdr.LedgerCloseMeta) error {
 	var txReader *ingest.LedgerTransactionReader
 	txReader, err := ingest.NewLedgerTransactionReaderFromLedgerCloseMeta(eventHandler.passphrase, lcm)
 	if err != nil {
-		return errors.Wrapf(err,
-			"failed to open transaction reader for ledger %d",
-			lcm.LedgerSequence())
+		return fmt.Errorf(
+			"failed to open transaction reader for ledger %d: %w ",
+			lcm.LedgerSequence(), err)
```

Comment:
why not wrap?

Reply:
oh well now I remember linter complains when I use `errors.Wrapf ` :|

`use of `errors.Wrapf` forbidden because "Do not use stellar/go/support/errors, use the standard 'errors' package and fmt.Errorf()." (forbidigo)`